js/ug_onEachFeature_fns.js

```diff
@@ -132,10 +132,10 @@ function prisons_OEF(layer) {
 
 function gdp_OEF(layer) {
     layer.bindPopup(
-        '<strong>District: </strong> ' + layer.feature.properties.DNama2017 +
```

Comment:
these spaces i put after the colons were there on purpose

Reply:
I do not see the difference. On the GDP Ug map for example. Check on the map that you have on this branch and what is currently in the master branch. Share a screenshot on here if possible